Intro to open source with GitHub/README.md

```diff
@@ -0,0 +1,22 @@
+# How to work on opensource contribution with Git
+
+## Making a contribution
+1. Fork the repo which you want to work on (eg. https://github.com/take2rohit/sih_number_plate is original repo where you want to contribute) 
+1. Clone the forked repository (not the above repo link)
+`git clone https://github.com/your_userid/sih_number_plate`
+3. Do your changes whatever you wish and commit 
+`git add .`
+`git commit -m "Adding my contribution"`
+4. Push changes to GitHub. This updates your forked repo.
+`git push origin master`
+5. Create a pull request
```

Comment:
maybe some website link which shows the process. I know it's not difficult, but is it easy for new people?

Reply:
will add